pika/adapters/libev_connection.py

```diff
@@ -284,9 +288,13 @@ def remove_timeout(self, timer):
 
         """
         LOGGER.debug('stop')
-        self._active_timers.pop(timer, None)
-        timer.stop()
-        self._stopped_timers.append(timer)
```

Comment:
this old code could lead to the same timer ending up multiple times on `self._stopped_timers` deque if user removed it multiple times or if user removed it after the timer callback was called (which also removes it)